server/util-server.js

```diff
@@ -353,6 +354,29 @@ exports.radius = function (
         ],
     });
 };
+/**
```

Comment:
```suggestion

/**
```
Preserve line between function and previous statement.